### PR TITLE
in_tail: Add option file_cache_advise

### PR DESCRIPTION
**Pull request by drbugfinder-work**
https://github.com/fluent/fluent-bit/pull/8422

### CODE REVIEW

File: pipeline/inputs/tail.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The plugin supports the following configuration parameters:
...
 | Static\_Batch\_Size | Set the maximum number of bytes to process per iteration for the monitored static files (files that already exists upon Fluent Bit start).                                                                                                                                                                                                                                                                                                                                                               | 50M     |
-
+| File\_Cache\_Advise | Set the posix_fadvise in POSIX_FADV_DONTNEED mode. This will reduce the usage of the kernel file cache. This option is only available on Linux.                                                                                                                                                                                                                                                                                                                                                              | On     |
```
Comment (patrick-stephens) [inline pipeline/inputs/tail.md:37]: By only available do you mean configuring the option causes a failure on non-Linux or just it does nothing?

Comment (drbugfinder-work) [reply pipeline/inputs/tail.md:37]: It just does nothing
https://github.com/fluent/fluent-bit/blob/8542fac9c29750818324fee1a6ce1a800e6c82c6/plugins/in_tail/tail_file.c#L1438-1445

Comment (patrick-stephens) [reply pipeline/inputs/tail.md:37]: OK, maybe reword then to say `will be ignored if not on Linux`?

Comment (drbugfinder-work) [reply pipeline/inputs/tail.md]: done.

**Review by lecaros — APPROVED** [review]
